src/main/java/snyk/analytics/Itly.java

```diff
@@ -125,45 +127,65 @@ public void identify(String userId, Identify properties) {
     public void group(String userId, String groupId, Group properties) {
         itly.group(userId, groupId, properties);
     }
-    /**
-     * Triggered when the analysis is loaded within the IDE.
-     *
-     * Owner: Georgi Mitev
-     * @param userId The user's ID.
-     * @param event
-     */
-    public void analysisIsReady(String userId, AnalysisIsReady event) {
-        track(userId, event);
-    }
-    /**
-     * User triggers an analysis or analysis is automatically triggered.
-     *
-     * Owner: Georgi Mitev
-     * @param userId The user's ID.
-     * @param event
-     */
-    public void analysisIsTriggered(String userId, AnalysisIsTriggered event) {
-        track(userId, event);
-    }
-    /**
-     * Triggered when the user clicks the health score of a package.
-     *
-     * Owner: Georgi Mitev
-     * @param userId The user's ID.
-     * @param event
-     */
-    public void healthScoreIsClicked(String userId, HealthScoreIsClicked event) {
-        track(userId, event);
-    }
-    /**
-     * Triggered when the user selects an issue from the issues list and the issue is loaded.
-     *
-     * Owner: Georgi Mitev
-     * @param userId The user's ID.
-     * @param event
-     */
-    public void issueInTreeIsClicked(String userId, IssueInTreeIsClicked event) {
-        track(userId, event);
+
+  /**
+   * Triggered when the analysis is loaded within the IDE.
+   * <p>
+   * Owner: Georgi Mitev
+   *
+   * @param userId The user's ID.
+   * @param event
+   */
+  public void analysisIsReady(String userId, AnalysisIsReady event) {
+    track(userId, event);
+  }
+
+  /**
+   * User triggers an analysis or analysis is automatically triggered.
+   * <p>
+   * Owner: Georgi Mitev
+   *
+   * @param userId The user's ID.
+   * @param event
+   */
+  public void analysisIsTriggered(String userId, AnalysisIsTriggered event) {
+    track(userId, event);
+  }
+
+  /**
+   * This Event fires when the authenticate button is clicked.
+   * <p>
+   * Owner: Bastian Doetsch
+   *
+   * @param userId The user's ID.
+   * @param event
+   */
+  public void authenticateButtonIsClicked(String userId, AuthenticateButtonIsClicked event) {
+    track(userId, event);
+  }
+
+  /**
+   * Triggered when the user clicks the health score of a package.
+   * <p>
+   * Owner: Georgi Mitev
+   *
+   * @param userId The user's ID.
+   * @param event
+   */
+  public void healthScoreIsClicked(String userId, HealthScoreIsClicked event) {
+    track(userId, event);
+  }
+
+  /**
+   * Triggered when the user selects an issue from the issues list and the issue is loaded.
+   * <p>
+   * Owner: Georgi Mitev
+   *
+   * @param userId The user's ID.
+   * @param event
+   */
+  public void issueInTreeIsClicked(String userId, IssueInTreeIsClicked event) {
+    track(userId, event);
```

Comment:
seems like reformating of ganerated code

Reply:
Yeah, happens because I had to manually change those events, as the generator is broken